File: Codigo/Obligatorio/codigo/ArenaGestorFront/src/app/models/Bands/BandResultBandDto.ts
import { BandResultArtistDto } from "./BandResultArtistDto";
import { BandResultGenderDto } from "./BandResultGenderDto";
import { BandResultConcertDto } from "./BandResultConcertDto";

export class BandResultBandDto {
    musicalProtagonistId: Number = 0;
    name: String = "";
    startDate: Date = new Date();
    gender: BandResultGenderDto = new BandResultGenderDto();
    artists: Array<BandResultArtistDto> = new Array();
    concerts: Array<BandResultConcertDto> = new Array();
}

File: Codigo/Obligatorio/codigo/ArenaGestor/ArenaGestor.APIContracts/Band/BandResultBandDto.cs
using System;
using System.Collections.Generic;

namespace ArenaGestor.APIContracts.Band
{
    public class BandResultBandDto
    {
        public int MusicalProtagonistId { get; set; }

        public string Name { get; set; }

        public DateTime StartDate { get; set; }

        public BandResultGenderDto Gender { get; set; }

        public List<BandResultArtistDto> Artists { get; set; }

        public List<BandResultConcertDto> Concerts { get; set; }
    }
}
